client/app/components/dashboards/dashboard-widget/VisualizationWidget.jsx

```diff
@@ -180,18 +182,18 @@ function VisualizationWidgetFooter({ widget, isPublic, onRefresh, onExpand }) {
       </span>
       <span>
         {!isPublic && (
-          <a
+          <PlainButton
             className="btn btn-sm btn-default hidden-print btn-transparent btn__refresh"
             onClick={() => refreshWidget(2)}>
             <i className={cx("zmdi zmdi-refresh", { "zmdi-hc-spin": refreshClickButtonId === 2 })} aria-hidden="true" />
             <span className="sr-only">
               {refreshClickButtonId === 2 ? "Refreshing, please wait." : "Press to refresh."}
             </span>
-          </a>
+          </PlainButton>
         )}
-        <a className="btn btn-sm btn-default hidden-print btn-transparent btn__refresh" onClick={onExpand}>
+        <PlainButton className="btn btn-sm btn-default hidden-print btn-transparent btn__refresh" onClick={onExpand}>
```

Comment:
Any chance we can just use a normal button, here?

Reply:
You mean a `<button>`? If that's the case, I'm trying to avoid it to keep consistency in the codebase. `<PlainButton>`'s only difference to it is the pointer.

Reply:
I actually just meant a `Button`